src/components/Map.ts

```diff
@@ -255,7 +278,7 @@ export class Map extends Component<MapProps, MapState> {
                         lng: results[0].geometry.location.lng()
                     });
                 } else if (status === google.maps.GeocoderStatus.OVER_QUERY_LIMIT) {
-                    this.setState({ alertMessage: `Google free quota request exceeded.` });
+                    this.setState({ alertMessage: "Google free quota request exceeded." });
```

Comment:
Improve massage, suggestion: `Failed to look up the address. Google Geocoder free request quota exceeded.`

Reply:
Done